cf/crypto.m4

```diff
@@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
 
 fi
 
+AC_ARG_WITH(pkcs11-module,
```

Comment:
Why shouldn't this just be a configuration parameter in krb5.conf?  Why bother with autoconf'ing this?  My guess: to provide a platform-specific/sane default?